lib/cylc/cfgspec/utils.py

```diff
@@ -17,37 +17,50 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """Utilities for configuration settings that are time intervals."""
 
+import re
 from isodatetime.data import Calendar
 from isodatetime.parsers import DurationParser
 from parsec.validate import (
     _strip_and_unquote, _strip_and_unquote_list, _expand_list,
-    IllegalValueError
-)
+    IllegalValueError)
+from cylc.mp_pool import SuiteFuncContext
```

Comment:
Awful name `SuiteFuncContext`. Probably my fault with `SuiteProcContext`!

Reply:
Agreed, and yes, just following the precedent!